tensorflow/tools/ci_build/windows/libtensorflow_cpu.sh

```diff
@@ -76,14 +76,12 @@ zip -j ${DIR}/libtensorflow_jni-cpu-windows-$(uname -m).zip \
 rm -f ${DIR}/tensorflow_jni.dll
 
 # Zip up the .dll, LICENSE and include files for the C library.
+mkdir -p ${DIR}/include/tensorflow/c
+mkdir -p ${DIR}/lib
+cp bazel-bin/tensorflow/libtensorflow.so ${DIR}/lib/tensorflow.dll
```

Comment:
we have another relative address in the source here. Do we also need to fix this?